research/src/pages/toolbar.proposal.mdx

```diff
@@ -0,0 +1,72 @@
+---
+menu: Proposals
+name: Toolbar (Editor's Draft)
+path: /components/toolbar
+pathToResearch: /components/toolbar.research
+---
+
+import ToolbaAnatomy from '../components/toolbar-anatomy'
+
+## Overview
+
+A `toolbar` is a container for grouping a set of controls, such as buttons, menubuttons, or checkboxes.
+
+## Use Cases
+
+A `Toolbar` groups visually a set of controls semantically algined and it can also be ane effective way of reducing the number of tab stops
+
+## Prior Art/Examples <a href="#prior-art" id="prior-art"></a>
+
+- [Stardust](https://fluentsite.z22.web.core.windows.net/0.59.0/components/toolbar/definition)
+- [Reakit](https://reakit.io/docs/toolbar/)
+- [Radix UI](https://www.radix-ui.com/docs/primitives/components/toolbar)
+
+### Anatomy <a class="link" href="#anatomy" id="anatomy"></a>
+
+<ToolbaAnatomy />
+
+#### Structure <a class="link" href="#structure" id="structure"></a>
+
+- `<toolbar>` - The root element that that has the semantic value of a toolbar **[required]**
+- `<toolbar-item>` - The item to be rendered in the toolbar **[required]**
+
+Proposed HTML:
+
+```html
+<div>
+  <button />
+  <div>
+    <button />
+    <button />
+  </div>
+  <span />
+  <button />
+</div>
```

Comment:
I'd probably prefer to understand the semantic representation of the structure here. Are the buttons here intended to be examples of slotted content? Similar to @scottaohara's question, what's the reason for the structure here? What are the named parts and why? In FAST, we identified a common pattern of having "start" and "end" content relating to layout, etc...that may not make sense here, but I think at the very least if there is additional DOM, understanding its intent is important.

Reply:
agreed.  there a disconnect between what this snippet is meant to represent, particularly in comparison to what the accessibility requirements are saying are necessary, as well.